tailscale.sh

```diff
@@ -31,67 +31,93 @@ curl -s "https://pkgs.tailscale.com/stable/${tarball}" -o tailscale.tgz
 
 echo "done."
 
+echo -n "Removing Legacy Installations..."
+
+# Stop and disable the systemd service
+if systemctl is-active --quiet tailscaled; then
+  systemctl stop tailscaled &>/dev/null || echo "ERROR: could not stop tailscaled"
+fi
+if systemctl is-enabled --quiet tailscaled; then
+  systemctl disable tailscaled &>/dev/null || echo "ERROR: could not disable tailscaled"
+fi
+
+# Remove the systemd system extension
+if [ $(systemd-sysext list 2>/dev/null | grep -c "/var/lib/extensions/tailscale") -ne 0 ]; then
+  systemd-sysext unmerge &>/dev/null || echo "ERROR: could not unmerge system extensions"
+  rm -rf /var/lib/extensions/tailscale
+  systemd-sysext merge &>/dev/null || echo "ERROR: could not merge system extensions"
+fi
+
+# Remove the overrides conf
+if test -f /etc/systemd/system/tailscaled.service.d/override.conf; then
+  rm -rf /etc/systemd/system/tailscaled.service.d/override.conf
```

Comment:
Is there some config here that we'd like to preserve across updates? I don't have my Steam Deck with me at the moment, but I'll check it as soon as I can and report back

Reply:
This is a good point, we should not blindly remove this as it is a valid path for overriding values in the unit file

Reply:
I took a look and this file hasn't changed since installation, it's safe to say we fully control it so it can be deleted